### PR TITLE
Implement Stats command

### PR DESCRIPTION
**Pull request by moyj01**

### CODE REVIEW

File: src/main/java/seedu/flashcard/ui/MainWindow.java [hunk 1]
```diff
@@ -241,7 +241,11 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 enterStudyMode(new ReviewPanel(logic, this));
             }
 
-            if (commandResult.getViewIndex() != null) {
+            if (commandResult.isShowStats() && commandResult.getViewIndex() != null) {
+                //TODO
```
Comment (tan-zhuoyao) [inline src/main/java/seedu/flashcard/ui/MainWindow.java:245]: Is this if block left blank intentionally?

**Review by tan-zhuoyao — COMMENTED** [review]
Please refer to my 1 comment, thank you!

File: src/main/java/seedu/flashcard/logic/commands/StatsCommand.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class StatsCommand extends Command {
+
+    public static final String COMMAND_WORD = "stats";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Shows the flashcard's statistics identified by the index number used in the displayed "
+            + "flashcard list.\n"
+            + "Parameters: INDEX (must be a positive integer)"
+            + " [" + PREFIX_ANSWER + "]\n"
+            + "Example: " + COMMAND_WORD + " 1";
```
Comment (sgn00) [inline src/main/java/seedu/flashcard/logic/commands/StatsCommand.java]: statscommand shldnt have any flags

**Review by sgn00 — CHANGES_REQUESTED** [review]
Fix the change

File: src/main/java/seedu/flashcard/ui/MainWindow.java (hunk 1 above)
Comment (moyj01) [reply src/main/java/seedu/flashcard/ui/MainWindow.java:245]: @sgn00 will implement this in the UI

**Pull request merged**